admin/serverAdmin/webserverAdmin.go

```diff
@@ -0,0 +1,469 @@
+package serverAdmin
+
+import(
+	"net/http"
+	"fmt"
+	"io/ioutil"
+	"html/template"
+	"strings"
+	"os"
+	"path"
+	"time"
+	"encoding/json"
+	"path/filepath"
+	"strconv"
+	"crypto/rand"
+	"io"
+	"encoding/base64"
+)
+
+var MainStaticRoot string = "../static"
+
+var StaticRoot string = "./static"
+
+var dirGroups string = "../groups/"
+
+var globalUser map[string]*TimeSession = make(map[string]*TimeSession)
+
+var cookie_token string = "session_token"
+
+type TimeSession struct {
+	LastAccess	time.Time
+	Expiration	time.Duration
+}
+
+type User struct {
+	Username	string
+	Password	string
+}
+
+type Config struct {
+	Admin	[]User
+}
+
+type Form struct {
+    Name	string
+	Error	string
+}
+
+type FileJson struct {
```

Comment:
Maybe we can make <https://github.com/jech/galene/blob/master/group/group.go#L701> public to remove duplicated code?

Reply:
I change my code to correspond with the struct defined by Galène, but I need to have a public access to the struct and the `fileName` attribute. With this we could delete my redefinition and it should works.

Reply:
No problem, I'll do that as soon as I create a 0.4 branch.

Reply:
@Aeljs why do you need the `fileName` field to be public?  Be aware that it has strange semantics (it's the filename from which the group configuration was read, so it's not necessarily related to the name of the group, the group might be an automatically created subgroup), so I consider it as an internal implementation detail.  No problem if you insist it should be exported, but we need to make sure you're aware of its meaning.

Reply:
I need the fileName field to be public to be able to display it on the interface. Currently I only use to list the different existing groups, and to create more if wanted. 
Like that:
![galene_admin](https://user-images.githubusercontent.com/44366999/115709536-a0d6dd80-a371-11eb-932b-526e9ffc1ea8.png)

Reply:
Ok, exported as you requested.